.github/workflows/test-xamarinandroid.yml

```diff
@@ -17,14 +17,22 @@ name: test-xamarinandroid
         required: false
       AtlasPrivateKey:
         required: false
+      AWS_DEVICEFARM_ACCESS_KEY_ID:
+        required: true
+      AWS_DEVICEFARM_SECRET_ACCESS_KEY:
+        required: true
+      DEVICEFARM_PROJECT_ARN:
+        required: true
+      DEVICEFARM_ANDROID_POOL_ARN:
+        required: true
 env:
   REALM_DISABLE_ANALYTICS: true
   DOTNET_NOLOGO: true
 jobs:
   run-tests:
-    runs-on: macos-latest
+    runs-on: windows-latest
```

Comment:
Can this run on Linux instead?

Reply:
We can't build Xamarin.Android projects on Linux.